structs/student_validator.c

```diff
@@ -0,0 +1,46 @@
+#include<stdio.h>
+
+typedef struct 
+{
+    int age;
+    int birth;
```

Comment:
You were meant to use `Date` type (struct) for the `date_of_birth` field of `StudentDate`. Not a regular `int`

Reply:
Done